odoo/addons/base/models/ir_embedded_actions.py

```diff
@@ -75,7 +75,8 @@ def _compute_is_visible(self):
             active_model_record = self.env[parent_res_model].search(domain_id, order='id')
             for record in records:
                 action_groups = record.groups_ids
-                if not action_groups or (action_groups & self.env.user.all_group_ids):
+                python_method_exists = not record.python_method or hasattr(self.env[parent_res_model], str(record.python_method))
+                if python_method_exists and (not action_groups or (action_groups & self.env.user.all_group_ids)):
```

Comment:
```suggestion
                python_method_exists = record.python_method and hasattr(self.env[parent_res_model], str(record.python_method))
                if python_method_exists and (not action_groups or (action_groups & self.env.user.all_group_ids)):
```

You should retarget this branch to 18.0 since the error occurs there.

In SaaS-17.4, this error is not occurring due to the embed view being unlinked during module uninstall. Could you identify why it is not deleted from 18.0?

Reply:
In **SaaS-17.4**, we use `ir.actions.act_window` in action_id :- https://github.com/odoo/odoo/blob/d5baeec9b60bdf3a5ab9d1243f46e957c0489877/odoo/addons/base/models/ir_embedded_actions.py#L19

And in **18.0**, we change it to `ir.actions.actions`:-
https://github.com/odoo/odoo/blob/e7efc2c3ad70aa9273412b089a71a9ade75ad525/odoo/addons/base/models/ir_embedded_actions.py#L19

Also, use `ondelete="cascade"` in both the versions. but `ondelete="cascade"` not works on `ir.actions.actions`.
https://github.com/odoo/odoo/blob/e7efc2c3ad70aa9273412b089a71a9ade75ad525/odoo/addons/base/models/ir_actions.py#L117-L127

We should remove `ondelete="cascade"` from the `action_id` field and add explicit unlink logic for `ir.embedded.actions`in the unlink method.

I have included these changes in this PR.

Thanks :slightly_smiling_face: